### PR TITLE
[14.0][FIX+IMP] product_brand_tag

### PR DESCRIPTION
**Pull request by ivantodorovich**
Form when using the create & edit m2m widget option

Before:
before

After:
after

### CODE REVIEW

File: product_brand_tag/models/product_brand_tag_mixin.py [hunk 1]
```diff
@@ -59,22 +58,18 @@ def _compute_brands_count(self):
...
-                (
-                    AsIs(table),
-                    tuple(self.ids),
-                ),
+                SELECT {column1}, COUNT(*)
+                FROM {relation}
+                WHERE {column1} IN %s
+                GROUP BY {column1}
+                """
+            ).format(
```
Comment (simahawk) [inline product_brand_tag/models/product_brand_tag_mixin.py:69]: FTR you can pass kw args even w/out using `sql.*` features. Just pass a dict to `execute`

File: product_brand_tag/models/product_brand_tag.py [hunk 2]
```diff
@@ -2,10 +2,12 @@
...
-from odoo import models
+from odoo import fields, models
 
 
 class ProductBrandTag(models.Model):
     _name = "product.brand.tag"
     _inherit = "product.brand.tag.mixin"
     _description = "Product Brand Tag"
+
+    product_brand_ids = fields.Many2many(relation="product_brand_tag_rel")
```
Comment (simahawk) [inline product_brand_tag/models/product_brand_tag.py:13]: I did it in extending modules only https://github.com/OCA/brand/blob/14.0/product_brand_social_responsibility/models/product_brand_tag_csr.py but definitely better to do it here as well :+1:

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@sbejaoui cound you have a look here?

**Review by sbejaoui — APPROVED** [review]
LGTM, thank you

**Pull request merged**